test/curry3.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('curry3', function() {
+
+  eq(typeof S.curry3, 'function');
+  eq(S.curry3.length, 4);
+
+  var curried = S.curry3(function(x, y, z) { return x * y * z; });
+  eq(curried(1, 2, 3), 6);
+  eq(curried(1)(2)(3), 6);
+  eq(curried(1, 2)(3), 6);
+  eq(curried(1)(2, 3), 6);
```

Comment:
For consistency, let's use the script to generate the assertions for `curry2` and `curry3` as well. You have all the combinations, but my obsessive streak wants to see them consistently ordered. :stuck_out_tongue_winking_eye: